### ISSUE TITLE
IntroVAE study

### ISSUE THREAD

**Issue by AlexLacson** [LindsayXX/NEM#5]
- [x] Study network architecture

- [x] Training procedure

- [x] Gradients and backprop procedure

- [x] Parameter settings

- [x] Optimization methods

- [x] Data preprocess/augmentation
